Cargo.toml

```diff
@@ -236,6 +236,7 @@ pretty-hex = "0.3.0"
 proc-macro2 = "1.0"
 progenitor = { git = "https://github.com/oxidecomputer/progenitor", branch = "main" }
 progenitor-client = { git = "https://github.com/oxidecomputer/progenitor", branch = "main" }
+bhyve_api = { git = "https://github.com/oxidecomputer/propolis", rev = "db39a40be172109de3ca34af8a1c37dd76e715a9" }
```

Comment:
Perhaps it's time to split this off into it's own home. Though, we're gonna be packaging together the OS/bhyve, propolis & sled-agent so maybe we should just make sure to keep all the propolis stuff on the same rev. Not really possibly yet since omicron is still pinned on an older rev but in the future. Anyways, not a blocker for this PR.